Security:  Fix cross site scripting vulnerability in CGI.pm (Bug#202367).

diff -ur perl-5.6.1.orig/lib/CGI.pm perl-5.6.1/lib/CGI.pm
--- perl-5.6.1.orig/lib/CGI.pm	Sun Mar  4 06:53:20 2001
+++ perl-5.6.1/lib/CGI.pm	Tue Jul 29 22:51:04 2003
@@ -1500,6 +1500,7 @@
        $action = $self->url(-absolute=>1,-path=>1);
        $action .= "?$ENV{QUERY_STRING}" if $ENV{QUERY_STRING};
     }
+    $action =~ s/\"/%22/g;  # fix cross-site scripting bug reported by obscure
     $action = qq(action="$action");
     my($other) = @other ? " @other" : '';
     $self->{'.parametersToAdd'}={};
